smartsheet-csharp-sdk/main/Smartsheet/Api/FolderSheetResources.cs

```diff
@@ -77,7 +77,7 @@ public interface FolderSheetResources
         /// <exception cref="ResourceNotFoundException"> if the resource cannot be found </exception>
         /// <exception cref="ServiceUnavailableException"> if the REST API service is not available (possibly due to rate limiting) </exception>
         /// <exception cref="SmartsheetException"> if there is any other error during the operation </exception>
-        Sheet ImportCsvSheet(long folderId, string file, string sheetName, int? headerRowIndex, int? primaryColumnIndex);
+        Sheet ImportCsvSheet(long folderId, string file, string? sheetName, int? headerRowIndex, int? primaryColumnIndex);
```

Comment:
<img width="2547" alt="Screenshot 2024-04-30 at 8 43 30 AM" src="https://github.com/smartsheet/smartsheet-csharp-sdk/assets/8115836/fbe589e5-b5bd-4709-b49b-6b2ca7b524c1">

There is a need to update the public API docs. Issue listed here: [#61](https://github.com/smartsheet/smartsheet-csharp-sdk/issues/61)